### DISCUSSION TITLE
Sync Problem Thunderbird

### DISCUSSION THREAD

**Discussion by jfabini** [kewisch/gdata-provider#592] [Support]
I've a sync issue between the Google Calendar and Thunderbird 102.10.0 and GProvider 102.0.3  on Linux Mint 21.1 Cinnamon. A calendar item that originally was created in TB has been synced to Google and modified later on several time (using Google's Web UI and several other Calendar clients).
However, the latest changes on the server for this specific item are not reflected in TB. I've tried to force synchronization, to stop and restart TB, rebooted the system: the particular TB calendar entry continues to display old values. 
Even more strange: 
1. If I hover with the mouse cursor over the TB calendar entry, the yellow overview window shows the correct entry, including the additions that are missing when opening/attempting to edit the item in TB! Seems like TB/GCal cache some values and do not update them. 
2. A second TB client running on another computer (Mint 20.3) shows identical behavior: it doesn't sync this specific entry
3. Other clients (iOS for instance) display the correct entry.  
4. I was not able to reproduce this issue for other entries (created a calendar entry in TB, synced it to Google, edited it... all changes are moved forth and back as it should).

I guess I could fix the issue by copying the content from the Goggle Web UI, pasting it into the calendar item on TB that is open for edit and forcing the sync back to Google. But for now I'll leave it as-is - perhaps there are some debugging options to identify the underlying reason why GCal/TB doesn't sync. Seems to be a bug that's worth fixing (after all it can lead to loss of information).

**Comment by martinmagnusson** [comment]
Do you think this is related to #497?  You could try the workaround mentioned there by jachoo: `await messenger.storage.local.set({"settings.idleTime": 0})`
